### PR TITLE
[Rados][TFA] Verification of OSD state with scrubbing post large OMAP warning

### PR DESCRIPTION
**Pull request by harshkumarRH**
[RHCEPHQE-11151](https://issues.redhat.com/browse/RHCEPHQE-11151): This is a change of test workflow for a TFA issue encountered with the test case - [CEPH-83572661](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83572661)

Minor changes to the workflow have already been made, local run from magna is also in-progress, awaiting pass logs to raise the PR
Recently we had made changes to the OMAP entries workflow to include OSD restarts, as we used to check OSD status concurrently with OMAP entries, there is a chance to find few OSDs not being up all the time, I have changed OSD status verification to run in parallel with scrubbing and deep-scrubbing only

Test modules -
- tests/rados/test_scrub_omap.py
- ceph/rados/pool_workflows.py

Pass logs:
Reef - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2ED3ND
Latest - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-C2Z6A7/

### CODE REVIEW

**Comment by neha-gangadhar** [comment]
LGTM

**Review by SrinivasaBharath — COMMENTED** [review]
Please delete the  commented code

**Review by SrinivasaBharath — COMMENTED** [review]
Please delete the  uncomment the code

**Comment by harshkumarRH** [comment]
Latest pass logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-C2Z6A7/
Removing DNM tag

File: tests/rados/test_scrub_omap.py [hunk 1]
```diff
@@ -43,70 +43,87 @@ def run(ceph_cluster, **kw):
...
-    omap_thread.daemon = True
-    omap_thread.start()
+    scrub_thread.daemon = True
+    log.info("Starting scrubbing thread")
+    scrub_thread.start()
+    log.info("Starting OSD state verification thread")
     verification_osd_thread.start()
     verification_osd_thread.join()
-    omap_thread._delete()
+    scrub_thread._delete()
```
Comment (pdhiran) [inline tests/rados/test_scrub_omap.py:80]: change this to use .join() instead of _delete().

Let's wait for scrub completion.

**Review by pdhiran — COMMENTED** [review]
Looks good with Very minor changes.

Please add pass logs post 1 run on reef.

File: tests/rados/test_scrub_omap.py (hunk 1 above)
Comment (harshkumarRH) [reply tests/rados/test_scrub_omap.py]: Changed, thanks for pointing out

**Comment by harshkumarRH** [comment]
Pass logs after changing scrub_thread._delete() to scrub_thread.join() - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-GGLSY1

**Review by pdhiran — APPROVED** [review]
Looks good

**Pull request merged**